home.md

```diff
@@ -30,6 +30,11 @@ iframe.twitter-follow-button {
 </style>
 <a href="https://twitter.com/virtualsciforum?ref_src=twsrc%5Etfw" class="twitter-follow-button" data-size="large" data-dnt="true" data-show-count="true">Follow @virtualsciforum</a><script async src="https://platform.twitter.com/widgets.js" charset="utf-8"></script>
 
+### Join us
+
+We are always happy to welcome new members on the organizational side of VSF. 
+Please take a look at our [call for organizers page](announcement.md) for more information.
```

Comment:
Please take a look [here](https://virtualscienceforum.org/#/contact) to see how to keep up with our activities or join our organising team.

Reply:
I think this is great - I just modified the sentence to link to the contact us page.